.github/workflows/ci-pipeline.yml

```diff
@@ -303,9 +303,63 @@ jobs:
           else
             echo "result=skip" >> $GITHUB_OUTPUT
           fi
-  vib-verify:
+  chart-score:
     runs-on: ubuntu-latest
     needs: [get-chart, update-pr]
```

Comment:
Could we remove the `update-pr` dependency? We are calculating the score form main branch so I think we can run `update-pr` and `chart-score` in parallel

Reply:
The reason why I chose to wait for `update-pr` is because the `chart-score` step does not make sense unless the `vib-verify` is executed right after, but I guess that by executing both in parallel we would be saving some extra time.

Reply:
Updated